tests/plugins/cloudservices/cloudservices.js

```diff
@@ -0,0 +1,122 @@
+/* @bender-ckeditor-plugins: cloudservices */
+( function() {
+	bender.editor = {
+		config: {
+			cloudServices_url: 'cs_url',
+			cloudServices_token: 'cs_token'
+		}
+	};
+
+	var mockBase64 = 'data:image/gif;base64,R0lGODlhDgAOAIAAAAAAAP///yH5BAAAAAAALAAAAAAOAA4AAAIMhI+py+0Po5y02qsKADs=';
+
+	bender.test( {
+		setUp: function() {
+			this.cloudservices = CKEDITOR.plugins.cloudservices;
+			this.editor.config.cloudServices_token = 'cs_token';
+			this.editor.config.cloudServices_url = 'cs_url';
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) {
```

Comment:
I think it should be `CKEDITOR.env.version < 10` as IE8/IE9 is not supported.

Reply:
You may also move this if to the beginning of `setUp` function so it can be catched on the first glimpse on the code.